e2e/suite_test.go

```diff
@@ -68,19 +68,42 @@ func waitKindnet() error {
 	return nil
 }
 
+func isDaemonsetLvmdEnvSet() bool {
+	return os.Getenv("DAEMONSET_LVMD") != ""
+}
+
+func skipTestIfNeeded() {
```

Comment:
```suggestion
func skipTestIfDaemonsetLvmd() {
```

`Needed` is too abstract.

Reply:
Would you add comments before using this function?
It may be `Skip because this test requires multiple nodes but there is just one node in daemonset lvmd test environment.`

Reply:
renamed and added coomments.